birthday/loop.py

```diff
@@ -79,10 +79,11 @@ async def _update_birthdays(self):
 
             birthday_members: dict[discord.Member, datetime.datetime] = {}
 
-            today_dt = datetime.datetime.utcnow().replace(
+            hour_td = datetime.timedelta(seconds=all_settings[guild.id]["time_utc_s"])
+
+            today_dt = (datetime.datetime.utcnow() - hour_td).replace(
                 hour=0, minute=0, second=0, microsecond=0
             )
-            hour_td = datetime.timedelta(seconds=all_settings[guild.id]["time_utc_s"])
 
             start = today_dt + hour_td
             end = start + datetime.timedelta(days=1)
```

Comment:
Are you sure about this logic? On line 84 the `.replace` replacing the hour makes the `- hour_td` have no effect, apart from potentially moving it back to yesterday if it is greater than 0?

Reply:
Wait, not quite. Let me try to wrap my head around this again.